source/common/http/http3/conn_pool.cc

```diff
@@ -59,10 +59,11 @@ allocateConnPool(Event::Dispatcher& dispatcher, Random::RandomGenerator& random_
                  Upstream::HostConstSharedPtr host, Upstream::ResourcePriority priority,
                  const Network::ConnectionSocket::OptionsSharedPtr& options,
                  const Network::TransportSocketOptionsConstSharedPtr& transport_socket_options,
-                 Upstream::ClusterConnectivityState& state, TimeSource& time_source) {
+                 Upstream::ClusterConnectivityState& state, TimeSource& time_source,
+                 Quic::QuicStatNames& quic_stat_names, Stats::Scope& scope) {
   return std::make_unique<Http3ConnPoolImpl>(
       host, priority, dispatcher, options, transport_socket_options, random_generator, state,
-      [](HttpConnPoolImplBase* pool) -> ::Envoy::ConnectionPool::ActiveClientPtr {
+      [&](HttpConnPoolImplBase* pool) -> ::Envoy::ConnectionPool::ActiveClientPtr {
```

Comment:
I think for non-test uses I'd avoid capturing `[&]` -- can you capture just what's needed?